ci_framework/roles/local_env_vm/tasks/main.yml

```diff
@@ -92,7 +92,8 @@
       --name "{{ cifmw_vm_name }}"
       --memory "{{ cifmw_local_env_vm_memory }}"
       --vcpus "{{ cifmw_local_env_vm_cpu }}"
-      --cpu host
+      --cpu host-passthrough
```

Comment:
host-passthrough always a good idea, unless you want to live migrate (to an older machine).